libs/data-mapper-v2/src/components/common/DropdownTree.tsx

```diff
@@ -0,0 +1,115 @@
+import { Tree, TreeItem, TreeItemLayout, Text } from '@fluentui/react-components';
+import { SearchBox } from '@fluentui/react';
+import { ChevronRightRegular, ChevronDownRegular } from '@fluentui/react-icons';
+import { useIntl } from 'react-intl';
+import type { ITreeItem } from 'models/Tree';
+import useStyles from './styles';
+import { useState, useMemo, useCallback } from 'react';
+import { isEmptyString } from '@microsoft/logic-apps-shared';
+
+interface DropdownTreeProps {
+  items: ITreeItem[];
+  onItemSelect: (item: ITreeItem) => void;
+}
+
+export const DropdownTree = (props: DropdownTreeProps) => {
+  const [showDropdownTree, setShowDropdownTree] = useState(false);
+  const [searchValue, setSearchValue] = useState('');
+
+  const intl = useIntl();
+  const styles = useStyles();
+
+  const selectSchema = intl.formatMessage({
+    defaultMessage: 'Select schema',
+    id: '3pheF6',
+    description: 'Select schema',
+  });
+
+  const search = intl.formatMessage({
+    defaultMessage: 'Search',
+    id: 'toWTrl',
+    description: 'Search from file list',
+  });
+
+  const onFileNameSelect = (item: ITreeItem) => {
+    props.onItemSelect(item);
+    setShowDropdownTree(false);
+    console.log(item.name);
```

Comment:
Nit: console.log

Reply:
Need that for the time being since onSelect isn't really doing anything but yes will remove this once I wire up the api call